packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverActions/renderTeachingPopoverActions.tsx

```diff
@@ -1,15 +1,13 @@
-/** @jsxRuntime automatic */
-/** @jsxImportSource @fluentui/react-jsx-runtime */
-
-import { assertSlots } from '@fluentui/react-utilities';
-import type { TeachingPopoverActionsState, TeachingPopoverActionsSlots } from './TeachingPopoverActions.types';
+import * as React from 'react';
+import type { TeachingPopoverActionsState } from './TeachingPopoverActions.types';
+import { getSlotsNext } from '@fluentui/react-utilities';
+import { TeachingPopoverActionsSlots } from './TeachingPopoverActions.types';
 
 /**
  * Render the final JSX of TeachingPopoverActions
  */
 export const renderTeachingPopoverActions_unstable = (state: TeachingPopoverActionsState) => {
-  assertSlots<TeachingPopoverActionsSlots>(state);
+  const { slots, slotProps } = getSlotsNext<TeachingPopoverActionsSlots>(state);
```

Comment:
This should use new JSX pragma & `assertSlots()`, check other components for examples.

Reply:
@behowell can you follow up and make sure our component creation guidance calls this out